db/migrate/20230210115830_add_qualified_teachers_api_status_and_sent_to_qualified_teachers_api_at_to_participant_outcomes.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddQualifiedTeachersApiStatusAndSentToQualifiedTeachersApiAtToParticipantOutcomes < ActiveRecord::Migration[6.1]
+  def change
+    add_column :participant_outcomes, :qualified_teachers_api_status, :boolean, null: false, default: false
+    add_column :participant_outcomes, :sent_to_qualified_teachers_api_at, :boolean, default: false
```

Comment:
could this column be set to a timestamp?

Reply:
all done 🤦‍♂️ tks @cwrw